css/details-styles.css

```diff
@@ -0,0 +1,98 @@
+* {
```

Comment:
this seems like alot of duplicate from the `styles.css` file, why not just include that one and add new stuff in there?